kuma/static/js/wiki-compat-tables.js

```diff
@@ -0,0 +1,274 @@
+                    $td = null;
+                }
+
+                // Close what's open, if anything, and open if $td has a value
+                hideHistory($td);
+            }
+
+            // Opens the history for a given item
+            function showHistory() {
+                if(animating) {
+                    return;
+                }
+
+                var $row = $openCell.closest('tr');
+                var $history = $openCell.find('.bc-history').outerWidth($table.width());
+
+                // get cell coords
+                var cellLeft = $openCell.offset().left;
+
+                // get cell left border
+                var cellLeftBorder = $openCell.css('border-left-width');
+
+                // get table coords
+                var tableLeft = $table.offset().left;
+
+                // left coord of table minus left coord of cell
+                var historyLeft = tableLeft - cellLeft - parseInt(cellLeftBorder, 10);
+
+                // get cell height
+                var cellTop = $openCell.outerHeight();
+
+                // get cell top border
+                var celltopBorder = $openCell.css('border-top-width');
+
+                // get cell bottom border
+                var cellBottomBorder = $openCell.css('border-bottom-width');
+
+                var historyTop = cellTop - parseInt(cellBottomBorder, 10) - parseInt(celltopBorder, 10);
+
+                var historyHeight;
+                var $subject;
+                var displayDetect;
+
+                // Add a close button if one doesn't already exist
+                if($history.find('.bc-history-button').length === 0) {
+                    $historyCloseButton.clone().appendTo($history);
+                }
+
+                // move history where it will display
+                $history.css({ left: historyLeft, top: historyTop });
+
+                // measure height
+                $history.css('display', 'block').attr('aria-hidden', false);
+                historyHeight = $history.outerHeight();
+
+                // set max-height to 0 and visibility to visible
+                $openCell.addClass('active').attr('aria-expanded', true);
+
+                // add measured height to history and to the cell/row it is being displayed beneath (CSS handles transition)
+                displayDetect = $table.find('thead td').first().css('z-index');
+
+                if(displayDetect === 1 ) {
+                    $subject = $row.find('td');
+                } else if(displayDetect === 2) {
+                    $subject = $openCell;
+                } else {
+                    $subject = $row.find('th, td');
+                }
+
+                animating = true;
+                $history.css('height', 0).stop().animate({ height: historyHeight }, animationProps);
+
+                $subject.stop().animate({ borderBottomWidth: historyHeight }, subjectAnimation);
+            }
+
+            // Hides the history dropdown for a given cell
```

Comment:
Nit: 

```
/** 
 * Hides the history dropdown for a given cell
 * @param {Object} $td - Hides the dropdown for this cell
 */
```

Reply:
I really dislike having to type the * on every line. Why do you suggest we use the JSDoc format?